woocommerce/woocommerce-ios
# [Design review] Show "Product" in the product form navigation bar title

Closes: #12213


## Description


- Added default product screen title

## Testing instructions


- Open products tab
- Select one of the products
- Check that product details screen title is "Product"

## Screenshots


![Simulator Screenshot - iPhone 15 Pro - 2024-03-05 at 16 07 58](https://github.com/woocommerce/woocommerce-ios/assets/6242034/3acd093a-f47f-4124-a0bb-5ff5b9c65648)
![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-03-05 at 16 05 50](https://github.com/woocommerce/woocommerce-ios/assets/6242034/c01b7672-8dbf-4e55-951b-0e21d21db87c)

---
